### PR TITLE
Update files to match output generated by minimal template command

### PR DESCRIPTION
**Pull request by yucheng11122017**
**What is the purpose of this pull request?**
Update files to latest files generated by markbind init as mentioned in [#1810](https://github.com/MarkBind/markbind/issues/1810)

**Overview of changes**
Files match the files generated by markbind init

**Anything you'd like to highlight/discuss:**
nil

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**

`
Update repo to match output of current version of init command --template 
minimal
`
Checklist:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [X] Linked all related issues
- [X] No unrelated changes

### CODE REVIEW

**Review by tlylt — CHANGES_REQUESTED** [review]
Hi @yucheng11122017, thanks for the work!

Could you help me double check all the files included again, and remove unnecessary ones?
I am seeing files like:
- .idea IDE files
- log files

being committed.

**Comment by yucheng11122017** [comment]
@tlylt Apologies, I must have missed it. I've removed them

**Comment by tlylt** [comment]
Thanks @yucheng11122017, LGTM.
P.S. I am waiting for write access to this repo before I can merge and also follow up with the Netlify issues.

**Review by tlylt — APPROVED** [review]

**Pull request merged**
